homeassistant/components/command_line/cover.py

```diff
@@ -131,7 +140,17 @@ def __init__(
         self._value_template = value_template
         self._timeout = timeout
         self._attr_unique_id = unique_id
-        self._attr_should_poll = bool(command_state)
+        self._scan_interval = scan_interval
+
+    async def async_added_to_hass(self) -> None:
+        """Call when entity about to be added to hass."""
+        await super().async_added_to_hass()
+        if self._command_state:
+            self.async_on_remove(
+                async_track_time_interval(
```

Comment:
The downside of async_track_time_interval is that it will keep firing the CLI command, even if the last one wasn’t finished. Polling will actually avoid calling update on an entity twice.